frontend/views/login.html

```diff
@@ -35,13 +35,18 @@
           <label for="password-s-confirm">Confirm-Password</label>
           <input class="form-control" id="password-s-confirm" type="password" ng-model="signup.passwordConfirmation">
         </div>
+        <div class="form-group" ng-class="{'has-error': signupError.name != null}">
```

Comment:
We probably want to switch these to using the <form-input> directive. I don't think it existed when I originally did the login page.